js/huobi.js

```diff
@@ -2792,9 +2886,9 @@ module.exports = class huobi extends Exchange {
             const entry = data[i];
             const currencyId = this.safeString (entry, 'currency');
             const code = this.safeCurrencyCode (currencyId);
-            this.options['networkChainIdsByNames'][code] = {};
             const chains = this.safeValue (entry, 'chains', []);
             const networks = {};
+            this.options['networkChainIdsByNames'][code] = {};
```

Comment:
@ttodua I know this is not the right place but can't highlight it now, but I just want to ask, aren't we missing the 'usdt' chain to represent the OMNI?
![image](https://user-images.githubusercontent.com/43336371/207664535-bd9021b4-1549-4b35-9608-a9a9ba017dc8.png)
![image](https://user-images.githubusercontent.com/43336371/207664575-b25f9157-973d-4fea-87d6-c2b4b732f12f.png)

Reply:
@carlosmiei no, there is no OMNI (at least, at this moment, might have been in the past).

you can see that https://api.huobi.pro/v2/reference/currencies and also when I visually check HUOBI withdraw page, there is also no "omni ( usdt )" ooption. there are only those 8 chains that you have screenshoted.

Reply:
@ttodua ok never mind then, probably docs are outdated, thanks for checking it